jwst/pipeline/calwebb_detector1.py

```diff
@@ -76,6 +76,13 @@ def process(self, input):
         # open the input as a RampModel
         input = datamodels.RampModel(input)
 
+        if self.ramp_fit.algorithm.upper() == "LIKELY":
+            ngroups = input.data.shape[1]
+            if ngroups < 4:
+                log.warning(f"For the LIKELY ramp fit algorithm NGROUPS must be"
+                            f" at least 4, but NGROUPS = {ngroups}.")
```

Comment:
Add another message here to make it clear that it's defaulting to OLS_C instead of LIKELY.